target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -162,6 +162,11 @@ dlink,dwr-116-a1|\
 mzk-ex300np)
 	set_wifi_led "$boardname:green:wifi"
 	;;
+dlink,dwr-921-c1)
+	set_wifi_led "$boardname:green:wifi"
+	ucidef_set_led_netdev "lan" "lan" "$boardname:green:lan" "br-lan"
+	ucidef_set_led_default "sigstrength" "Signal Strength" "$boardname:red:sigstrength" "0"
```

Comment:
Normally I've ignored it, but as you have to touch the file anyway: Can we use the green sigstrength led for diag? red leds are usually used for warning/error. For sure a matter of taste and nothing really critical.

Reply:
I personally prefer the red to keep the attention high during the sysupgrade operation.
But I change in green.

Bye.